sensor_msgs/PointCloud2 cloud_input
---
sensor_msgs/PointCloud2[] horizontal_planes
float32[] horizontal_plane_coefs
visualization_msgs/Marker[] horizontal_plane_bounding_boxes